.github/actions/cache/action.yml

```diff
@@ -25,6 +28,16 @@ runs:
         echo "Current workflow path is: ${MIDDLE}"
         echo "path=${MIDDLE}" >> "$GITHUB_OUTPUT"
       shell: bash
+    - name: Generate meson cache key
+      if: ${{ inputs.use-meson == 'true' }}
+      run: |
+        echo "MESON_KEY=`meson --version`" >> $GITHUB_ENV
+      shell: bash
+    - name: Generate empty meson cache key
+      if: ${{ inputs.use-meson != 'true' }}
+      run: |
+        echo "MESON_KEY=\"0\"" >> $GITHUB_ENV
```

Comment:
Optional: The quotes around 0 don't seem necessary, because `meson --version` cannot be 0. But the quotes do make the format different. I wonder if we can just leave the right-hand side empty:

```
      run: |
        echo "MESON_KEY=" >> $GITHUB_ENV
```

Reply:
Please ignore this comment. I didn't realize this comment was still posted. I thought it wasn't saved. Sorry!